app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -38,10 +38,9 @@ def openstack_handle(options = {})
       username = options[:user] || authentication_userid(options[:auth_type])
       password = options[:pass] || authentication_password(options[:auth_type])
 
-      vmdb_config = VMDB::Config.new("vmdb").config
       extra_options = {
-        :ssl_ca_file    => vmdb_config.fetch_path(:ssl, :ssl_ca_file),
-        :ssl_ca_path    => vmdb_config.fetch_path(:ssl, :ssl_ca_path),
+        :ssl_ca_file    => ::Settings.ssl.ssl_ca_file,
+        :ssl_ca_path    => ::Settings.ssl.ssl_ca_path,
```

Comment:
Let's keep it general for now, and when we have an actual use case for a per-provider setting we can add it.